go/service/gregor.go

```diff
@@ -842,38 +841,46 @@ func (g *gregorHandler) newChatActivity(ctx context.Context, m gregor.OutOfBandM
 	if m.Body() == nil {
 		return errors.New("gregor handler for chat.activity: nil message body")
 	}
-	body, err := jsonw.Unmarshal(m.Body().Bytes())
-	if err != nil {
-		return err
-	}
 
-	action, err := body.AtPath("action").GetString()
-	if err != nil {
-		return err
-	}
+	g.G().Log.Debug("ignoring newChatActivity OOBM, needs fix CORE-3789")
 
-	var activity keybase1.ChatActivity
+	return nil
 
-	switch action {
-	case "newMessage":
-		var boxed chat1.MessageBoxed
-		if err = body.AtPath("payload").UnmarshalAgain(&boxed); err != nil {
+	// TODO: this needs to use msgpack, not json see CORE-3789
```

Comment:
Yes, I added that ticket so this can be fixed.  It's not related to this ticket, just something I found while testing.